tests/conftest.py

```diff
@@ -30,52 +36,187 @@
 
 
 @pytest.fixture(scope="session")
-def engine():
-    """Test-session-wide database engine"""
+def pkg_file_root():
```

Comment:
That's a very elegant solution to the `pkg_resources` migration! We *should* eventually try to steer away from using file system paths, and use in memory package resources if possible. But this transitions away from the deprecated behaviour very nicely.